### ISSUE TITLE
Error while run the project

### ISSUE THREAD

**Issue by BriscoPeas93** [Eternali/custom_radio#4]
../../flutter/.pub-cache/hosted/pub.dartlang.org/custom_radio-0.1.2/lib/custom_radio.dart:102:36: Error: The argument type 'Null Function(Null)' can't be assigned to the parameter type 'dynamic Function(void)'.
[        ]         _controller.reverse().then((Null value) {

**Comment by luis-jaramillo** [comment]
Hi there ¿ Any solution ?

**Comment by atakangl** [comment]
I have same issue on iOS

**Comment by Omar1408mmz** [comment]
hi guys any solution for this  ??

**Comment by jitendra-negi** [comment]
Yes, I have wasted my couple of days to solve this issue but finally i found the solution: [check this solution](https://github.com/Eternali/custom_radio/issues/2#issuecomment-525233240)

**Comment by luis-jaramillo** [comment]
Open the file ../../flutter/.pub-cache/hosted/pub.dartlang.org/custom_radio-0.1.2/lib/custom_radio.dart and override _updateState method:

```
 void _updateState() {
    setState(() {
      if (widget.checked && _controller.status != AnimationStatus.completed) {
        _controller.forward();
      } else {
        // _controller.reverse().then((Null value) {
        //   setState(() {});
        // });
        _controller.reverse();
      }
    });
  }

```

[Comment](https://github.com/Eternali/custom_radio/issues/1#issuecomment-501947613 )
